config/go.yml

```diff
@@ -40,4 +40,14 @@ entries:
 
 - prefix: /
   replacement: http://geneontology.org/ontology/
+  
+# Terms for GO REFs
+# https://github.com/geneontology/go-site/blob/master/metadata/gorefs/goref-0000001.md
+# should point to
+# http://purl.obolibrary.org/obo/go/references/0000001
+- regex: ^/references/(\d+)$
```

Comment:
Confusingly unlike the others entry types, `regex` has to match the full path. It should be: `^/obo/go/references/(\d+)$`